### PR TITLE
February newsletter

### PR DESCRIPTION
**Pull request by bethaitman**

### CODE REVIEW

File: docs/blog/newsletter-february-2021.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+.. post:: February 03, 2021
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – February 2021
+#########################################
+
+Hey everyone! It's a month into 2021 already and the newsletter team is back in business. We've got a packed issue for you today, with topics ranging from screenshots to embedded code snippets to showing your value, so I'll get right on with it.
+
+Just one announcement for today: the East Coast Quorum meetup is holding its first virtual event this month! It's on "Lessons learned open-sourcing GitHub Docs", and it'll be on Monday 22nd February. If you're interested, `sign up to attend <https://www.meetup.com/virtual-write-the-docs-east-coast-quorum/events/276054186/>`__.
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2021.rst]: worth mentioning a time + timezone here probs?

File: docs/blog/newsletter-february-2021.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+So what's been happening on Slack?
+
+------------------------------------------
+Getting the right balance with screenshots
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2021.rst]: What about

```suggestion
Getting the balance right with screenshots
```

File: docs/blog/newsletter-february-2021.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+ 
+It seems that it’s often part of a tech writer’s job to manage up, teaching both your peers and higher-ups about the value that both you and docs bring. Not easy, but the book club has some tips!
+
+First up, behave like other, more-established roles. If they document their processes in a wiki, do the same. Announce big docs changes just as you’d announce product changes.
+
+Also, look at the interpersonal dynamics. Tech writing often needs a lot of input from other roles - if this is seen as a zero-sum exchange that doesn’t benefit the other person, it’s not gonna work. You can mitigate this by doing your research up front to show you respect their time; by seeking out opportunities to help them with other things, so you’re giving back; and by emphasising your common goals, so it’s not us vs them. See if you can make things easier for people who are helping you - a little bending the rules can be the right call sometimes.
+
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2021.rst:40]: Last sentence doesn't seem very clearly connected here.

**Review by plaindocs — APPROVED** [review]
Looks good to me!

File: docs/blog/newsletter-february-2021.rst [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+For more on screenshots in documentation, check out Steve Stegelin's `Graphic Content Warning: The Pros, Cons, and Alternatives to Screenshots <https://www.writethedocs.org/videos/portland/2018/graphic-content-warning-the-pros-cons-and-alternatives-to-screenshots-steve-stegelin/>`_ from Write the Docs Portland 2018 and Swapnil Ogale's `When bad screenshots happen to good writers <https://www.writethedocs.org/videos/eu/2016/when-bad-screenshots-happen-to-good-writers-swapnil-ogale/>`_ from WTD EU 2016.
+
+--------------------------------------
+Report from the book club: managing up
+--------------------------------------
+
+This winter, the `Write the Docs book club <https://app.slack.com/client/T0299N2DL/C7YJR1N02>`__ is reading and discussing their way through  Managing Writers by Richard Hamilton. The book covers a broad range of topics on the overall topic of managing docs projects and teams (both big and small). There’s been some fab discussion every week, so here’s a sample on the theme of showing your value and finding your place in the org.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-february-2021.rst]: ...topics on the overall topic...

Maybe "...topics on the overall subject..." instead?

**Review by hillaryfraley — APPROVED** [review]
👍

File: docs/blog/newsletter-february-2021.rst [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+.. post:: February 03, 2021
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – February 2021
+#########################################
+
+Hey everyone! It's a month into 2021 already and the newsletter team is back in business. We've got a packed issue for you today, with topics ranging from screenshots to embedded code snippets to showing your value, so I'll get right on with it.
+
+Just a couple announcements today: the East Coast Quorum meetup is holding its first virtual event this month! It's on "Lessons learned open-sourcing GitHub Docs", and it'll be on Monday 22nd February at 19:30 Eastern Time. If you're interested, `sign up to attend <https://www.meetup.com/virtual-write-the-docs-east-coast-quorum/events/276054186/>`__. We also have a proposed Write the Docs Enhancement Proposal (WEP) focused on Slack moderation. There is an `open pull request <https://github.com/writethedocs/weps/pull/4>`__ with the proposal for ``WEP2``, and the community feedback period lasts until March 5.
```
Comment (ericholscher) [inline docs/blog/newsletter-february-2021.rst]: Just a note -- I added the WEP here since it's ready to go out. /cc @bethaitman

Comment (bethaitman) [reply docs/blog/newsletter-february-2021.rst]: Cool, thanks for letting me know.

File: docs/blog/newsletter-february-2021.rst (hunk 2 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2021.rst]: Added

File: docs/blog/newsletter-february-2021.rst (hunk 4 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2021.rst]: D'oh. Fixed.

File: docs/blog/newsletter-february-2021.rst (hunk 3 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2021.rst:40]: You're right, Removed.

**Pull request merged**